airflow/cli/commands/task_command.py

```diff
@@ -407,18 +408,19 @@ def task_run(args, dag=None):
     # this should be last thing before running, to reduce likelihood of an open session
     # which can cause trouble if running process in a fork.
     settings.reconfigure_orm(disable_connection_pool=True)
-
+    ret = None
     try:
         if args.interactive:
-            _run_task_by_selected_method(args, dag, ti)
+            ret = _run_task_by_selected_method(args, dag, ti)
```

Comment:
I this this can just be

```python
return _run_task_by_selected_method(args, dag, ti)
```

Since the `finally` block is run before the return actually returns.